sql/core/src/test/scala/org/apache/spark/sql/DatasetUnpivotSuite.scala

```diff
@@ -310,9 +327,9 @@ class DatasetUnpivotSuite extends QueryTest
       errorClass = "UNPIVOT_VALUE_DATA_TYPE_MISMATCH",
       parameters = Map(
         "types" ->
-          (""""BIGINT" \(`long1#\d+L`, `long2#\d+L`\), """ +
-           """"INT" \(`int1#\d+`, `int2#\d+`, `int3#\d+`, ...\), """ +
-           """"STRING" \(`str1#\d+`\)""")),
+          (""""BIGINT" \(`long1`, `long2`\), """ +
+           """"INT" \(`int1`, `int2`, `int3`, ...\), """ +
+           """"STRING" \(`str1`\)""")),
       matchPVals = true)
```

Comment:
`matchPVals = true` is not needed if you are sure that regexp is not needed, here. Just in case, could you explain why did you replace the regexps.

Reply:
good spot, this is not needed anymore, fixed